Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  TuistAcceptanceFixtures.swift
+//  TuistAcceptanceTesting
+//
+//  Created by Daniele Formichelli on 20/12/23.
+//
+
+import Foundation
+
+public enum TuistAcceptanceFixtures {
+    case appWithBuildRules
```

Comment:
In a follow-up we can document this code and use Swift DocC to generate a reference that includes what every fixture is about :)

Reply:
👍